panda.config.ts

```diff
@@ -4,8 +4,10 @@ import { globalCss } from './theme/global/global';
 
 export default defineConfig({
   preflight: true,
-
-  include: ['./src/**/*.{js,jsx,ts,tsx}'],
+  include: [
```

Comment:
This is an important step it took time to figure out with help from [this video](https://www.youtube.com/live/fzVyQxVvf7o?si=sKLvd4GwzJTWc0Te)

We need to let panda know about our library. Otherwise the CSS doesn't get applied correctly and things look weird 
- icons don't have their size set
- inputs have their labels not aligned etc.